pkg/config/config_test.go

```diff
@@ -22,6 +22,7 @@ func TestSetLocalConfiguration(t *testing.T) {
 	maxCPUValue := "2"
 	minMemValue := "500M"
 	testValue := "test"
+	ports := "8080/TCP,45/UDP"
```

Comment:
we should have tests for all the other options. The whole switch case in `SetConfiguration` needs to be covered

Reply:
maybe in another PR?

Reply:
the scope of this seems like to be resolving the panic

Reply:
It is just resolving this one particular case.
It does nothing to prevent from happening this again with other option.

Reply:
done, added e2e test for each param